gpflow/likelihoods/scalar_continuous.py

```diff
@@ -36,61 +45,92 @@ class Gaussian(ScalarLikelihood):
     likelihood variance by default.
     """
 
-    DEFAULT_VARIANCE_LOWER_BOUND = 1e-6
-
     def __init__(
         self,
-        variance: float = 1.0,
-        variance_lower_bound: float = DEFAULT_VARIANCE_LOWER_BOUND,
+        variance: Optional[ConstantOrFunction] = None,
+        *,
+        scale: Optional[ConstantOrFunction] = None,
+        variance_lower_bound: float = DEFAULT_LOWER_BOUND,
```

Comment:
Would it make sense to leave the name as before, i.e. DEFAULT_VARIANCE_LOWER_BOUND?

Reply:
> Would it make sense to leave the name as before, i.e. DEFAULT_VARIANCE_LOWER_BOUND?

Well, below it's used for `scale_lower_bound` and `shape_lower_bound` as well, so `DEFAULT_VARIANCE_LOWER_BOUND` would be a slightly weird name. I suppose I can just have a lot of constants instead, if you like that better?

Reply:
> Why do we introduce different parameterisations for Gaussian?

If you want to parameterise the `Gaussian` with a function, then it's a lot more intuitive to use the `scale` as that is "linear". If you use the `variance` then you'd need a `sqrt` function to model linear noise, which is a bit weird.